src/commands/tag/analyze-missing.command.ts

```diff
@@ -0,0 +1,96 @@
+import { bold, Command } from "../../deps.ts";
+import { CmdGlobalOptions } from "../cmd-options.ts";
+import { setupLogger } from "../../logger.ts";
+import { verifyCliAvailability } from "../../init.ts";
+import { loadConfig } from "../../config/config.model.ts";
+import { MeshAdapterFactory } from "../../mesh/mesh-adapter.factory.ts";
+import { MeshTenant } from "../../mesh/mesh-tenant.model.ts";
+
+export const analyzeMissingTags = new Command()
```

Comment:
d: I like this pattern of splitting up the commands. Should probably adapt this for the other commands e.g. tenant as well.

Reply:
Yup I think that's a great idea. I'll create a ticket. Maybe I'll pick this up sometime in my free time as I love code that is properly structured :-D